lib/membrane_http_adaptive_stream/hls.ex

```diff
@@ -45,21 +45,34 @@ defmodule Membrane.HTTPAdaptiveStream.HLS do
   defp serialize_track(%Track{} = track) do
     use Ratio
 
-    target_duration = Ratio.ceil(track.target_segment_duration / Time.second()) |> trunc
+    target_duration = Ratio.ceil(track.target_segment_duration / Time.second()) |> trunc()
     media_sequence = track.current_seq_num - Enum.count(track.segments)
 
     """
     #EXTM3U
     #EXT-X-VERSION:#{@version}
     #EXT-X-TARGETDURATION:#{target_duration}
     #EXT-X-MEDIA-SEQUENCE:#{media_sequence}
+    #EXT-X-DISCONTINUITY-SEQUENCE:#{track.current_discontinuity_seq_num}
```

Comment:
Should we always include the discontinuity sequence or just when it happens for the first time in the playlist?

It can be skipped from the RFC
```
If the Media Playlist does not contain an EXT-X-DISCONTINUITY-
 SEQUENCE tag, then the Discontinuity Sequence Number of the first
 Media Segment in the Playlist SHALL be considered to be 0.

 The EXT-X-DISCONTINUITY-SEQUENCE tag MUST appear before the first
 Media Segment in the Playlist.

 The EXT-X-DISCONTINUITY-SEQUENCE tag MUST appear before any EXT-
 X-DISCONTINUITY tag.
```